pkg/raft/testdata/lazy_replication.txt

```diff
@@ -48,15 +48,16 @@ stabilize
 # Attempt to send a misaligned MsgApp. No-op.
 send-msg-app 1 to=2 lo=10 hi=13
 ----
-could not send MsgApp (10,13] to 2
```

Comment:
Could you update the `# comments` above this command and the one below it? Now, the first send is aligned, and the second one isn't. This looks fine.